docs/mivot/index.rst

```diff
@@ -116,16 +116,16 @@ using the ``dict`` property of ``MivotInstance``.
 
     DictUtils.print_pretty_json(mivot_object_dict)
```

Comment:
Absolutely unrelated to this PR, but this example is not copy pasteable (though it looks exactly as the other, functioning examples), here `mivot_object` is not defined.

Reply:
Python snippet completed